### PR TITLE
Implement `koji_build` trigger for `bodhi_update` job

### PR DESCRIPTION
**Pull request by nforro**
Fixes https://github.com/packit/packit-service/issues/2379.

I purposefully didn't implement removing sidetags from DB after a successful update creation because I want to see first how exactly the Bodhi workflow works - at which point is the sidetag removed from Koji etc. Maybe it would be enough to leave it to the periodic cleanup: https://github.com/packit/packit-service/issues/2412

### CODE REVIEW

File: packit_service/service/api/bodhi_updates.py [hunk 1]
```diff
@@ -77,7 +77,7 @@ def get(self, id):
             "status": update.status,
             "branch": update.target,
             "web_url": update.web_url,
-            "koji_nvr": update.koji_nvr,
+            "koji_nvrs": update.koji_nvrs,
```
Comment (lbarcziova) [inline packit_service/service/api/bodhi_updates.py:80]: this should be probably updated in dashboard code as well

File: packit_service/worker/handlers/koji.py [hunk 2]
```diff
@@ -436,13 +439,23 @@ def run(self) -> TaskResults:
                 continue
             for job in packages_config.get_job_views():
                 if (
-                    job.type == JobType.koji_build
+                    job.type in [JobType.koji_build, JobType.bodhi_update]
```
Comment (lbarcziova) [inline packit_service/worker/handlers/koji.py:442]:
```suggestion
                    job.type in (JobType.koji_build, JobType.bodhi_update)
```

**Review by lbarcziova — APPROVED** [review]
LGTM, let's test this! 

> Maybe it would be enough to leave it to the periodic cleanup: https://github.com/packit/packit-service/issues/2412

:+1: let's see the behaviour first

**Comment by nforro** [comment]
recheck

**Pull request merged**